.github/workflows/tests.yml

```diff
@@ -222,16 +237,23 @@ jobs:
           sudo df -h
           echo
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
-      - name: Setup FFmpeg
-        uses: Iamshankhadeep/setup-ffmpeg@v1.2
+      - if: matrix.os == 'ubuntu-latest'
+        name: Setup FFmpeg
+        uses: Iamshankhadeep/setup-ffmpeg@v1.1  # does not support arm64
```

Comment:
Why is arm64 support relevant here? Dunno if I'm missing some context that would be obvious to everyone with github actions experience or if this comment could be more useful, but either way I should probably know the answer.

Reply:
The `macos-latest` runner (`macos-14` and newer) are arm64 architecture. This particular action downloads pre-built binaries and the source they are using doesn't have Apple Silicon binaries. 

There might be some improvement to be made here, but this is maintaining the status quo of how our CI is currently set up.